dashboard/test/ui/features/acquisition_products/sign_up.feature

```diff
@@ -9,8 +8,7 @@ Scenario: Teacher can create an account with the new school association flow
   And I press keys "password" for element "#user_password"
   And I press keys "password" for element "#user_password_confirmation"
   And I press "#signup_form_submit" using jQuery
-  # Add the experiment flag
-  And I am on "http://studio.code.org/users/sign_up?enableExperiments=school_association_v2"
+  And I wait for 2 seconds
```

Comment:
nit: why do we need a 2 second wait? Could we wait for an element to be available instead?

Reply:
Yes definitely- was just taking the easier route because I knew the command would pass. Initially when I added the 'once I see it' I misused the command and was getting failing builds. But I think I have it edited correctly this time!